### ISSUE TITLE
JSDOM 10.0.0 Breaks Karma-jsdom-launcher

### ISSUE THREAD

**Issue by DaveMBush** [badeball/karma-jsdom-launcher#14]
Upgrading jsdom to 10.0.0 (released this morning) breaks karma-jsdom-launcher with "env is not a function"

**Comment by rzymek** [comment]
The exact error is
```
 TypeError: jsdom.env is not a function
    at jsdomBrowser._start (.../node_modules/karma-jsdom-launcher/index.js:9:11)
```

In 10.0.0 the [way to import jsdom](https://github.com/tmpvar/jsdom#basic-usage) is:
```js
const jsdom = require("jsdom");
const { JSDOM } = jsdom;
const dom = new JSDOM(``, {
  url: "https://example.org/",
  referrer: "https://example.com/",
  contentType: "text/html",
  userAgent: "Mellblomenator/9000",
  includeNodeLocations: true
});
```

**Comment by badeball** [comment]
Thanks for the heads up! I've released a version with support for jsdom-10.0.0.
